src/home_robot/home_robot/navigation_planner/discrete_planner.py

```diff
@@ -372,24 +437,57 @@ def _get_short_term_goal(
             # Set multi goal to the dilated goal map
             # We will now try to find a path to any of these spaces
             planner.set_multi_goal(dilated_goal_map, self.timestep)
+            # Set dilated goal map for use with simulation code - use this to compute closest goal
```

Comment:
dilated_goal_map can be moved inside the if condition, now that it is used only when `plan_to_dilated_goal=True`